src/labkey/Domain.ts

```diff
@@ -17,10 +17,27 @@ import { buildURL } from './ActionURL'
 import { request } from './Ajax'
 import { getCallbackWrapper, isString } from './Utils'
 
+// This is effectively GWTDomain
+export interface IDomainDesign {
```

Comment:
I prefer it as it indicates that this is strictly an interface. This is the long-standing pattern I've used in `labkey-api-js` throughout all of the modules since well before I knew anything of C#.

Reply:
I changed `IDomainDesign` to `DomainDesign` with 6df2d42.